docs/source/notes/amp_examples.rst

```diff
@@ -0,0 +1,158 @@
+.. _amp-examples:
+
+Automatic Mixed Precision Examples
+==================================
+
+.. currentmodule:: torch.cuda.amp
+
+.. contents:: :local:
+
+.. _gradient-scaling-examples:
+
+Gradient Scaling
+^^^^^^^^^^^^^^^^
+
+Gradient scaling helps ensure convergence when training with mixed precision,
```

Comment:
"Convergence" out of context.

Reply:
Also, that's not exactly what gradient scaling does. It does something else that is intended to help with convergence?

Reply:
Natalia: You say this is "explained" in the link, but the linked text doesn't mention "convergence."